projects/ad7768-evb/src.mk

```diff
@@ -47,6 +47,7 @@ SRC_DIRS += $(DRIVERS)/axi_core/axi_adc_core
 SRCS += $(DRIVERS)/api/no_os_gpio.c \
         $(DRIVERS)/api/no_os_i2c.c  \
         $(DRIVERS)/api/no_os_irq.c  \
+	$(DRIVERS)/api/no_os_timer.c \
```

Comment:
Is this required, since this project doesn't use timers? I guess `no_os_timer.h` is also included unnecessarily.